### PR TITLE
feat/footer september 23

### PR DESCRIPTION
**Pull request by ahmaxed**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: build/universal/trending/english.yaml [hunk 1]
```diff
@@ -1,60 +1,60 @@
...
+article19link: "https://www.freecodecamp.org/news/sql-convert-the-date-to-string-or-datetime-function-2/"
+article20title: "Rename Column in Pandas"
+article20link: "https://www.freecodecamp.org/news/how-to-rename-a-column-in-pandas/"
+article21title: "Delete a File in Python"
+article21link: "https://www.freecodecamp.org/news/how-to-delete-a-file-in-python-and-remove-a-directory-too/"
+article22title: "K-Nearest Neighbors Algo"
+article22link: "https://www.freecodecamp.org/news/k-nearest-neighbors-algorithm-classifiers-and-model-example/"
+article23title: "iferror Function in Excel"
+article23link: "https://www.freecodecamp.org/news/iferror-function-in-excel-example/"
+article24title: "Remove from String Python"
```
Comment (scissorsneedfoodtoo) [inline build/universal/trending/english.yaml:49]:
```suggestion
article24title: "Remove From String Python"
```

File: build/universal/trending/english.yaml [hunk 2]
```diff
@@ -1,60 +1,60 @@
...
+article7link: "https://www.freecodecamp.org/news/matplotlib-figure-size-change-plot-size-in-python/"
+article8title: "Database Normalization"
+article8link: "https://www.freecodecamp.org/news/database-normalization-1nf-2nf-3nf-table-examples/"
+article9title: "Nested Lists in Python"
+article9link: "https://www.freecodecamp.org/news/list-within-a-list-in-python-initialize-a-nested-list/"
+article10title: "Coalesce SQL"
+article10link: "https://www.freecodecamp.org/news/coalesce-sql-example-postgresql-and-sql-server-functions/"
+article11title: "Python join()"
+article11link: "https://www.freecodecamp.org/news/python-join-how-to-combine-a-list-into-a-string-in-python/"
+article12title: "JS Post Request"
```
Comment (scissorsneedfoodtoo) [inline build/universal/trending/english.yaml:25]:
```suggestion
article12title: "JS POST Request"
```

File: build/universal/trending/english.yaml [hunk 3]
```diff
@@ -1,60 +1,60 @@
...
+article25title: "Create a File in Terminal"
+article25link: "https://www.freecodecamp.org/news/how-to-make-a-file-in-linux-from-the-command-line-create-a-file-in-terminal/"
+article26title: "Clear Formatting in Excel"
+article26link: "https://www.freecodecamp.org/news/how-to-clear-formatting-in-excel/"
+article27title: "Accounting Num Format Excel"
+article27link: "https://www.freecodecamp.org/news/accounting-number-format-in-excel-how-to-apply-it-to-selected-cells/"
+article28title: "Check if File Exists Python"
+article28link: "https://www.freecodecamp.org/news/how-to-check-if-a-file-exists-in-python/"
+article29title: "Iterate Over Dict in Python"
+article29link: "https://www.freecodecamp.org/news/dictionary-iteration-in-python/"
```
Comment (scissorsneedfoodtoo) [inline build/universal/trending/english.yaml:60]: This should add the newline back to the end of the file and fix the lint test:

```suggestion
article29link: "https://www.freecodecamp.org/news/dictionary-iteration-in-python/"

```

**Review by scissorsneedfoodtoo — COMMENTED** [review]
Took a look and all of the links appear to work, and should be pointing to the correct articles.

Made a couple of suggestions for the titles, but they aren't necessary. The only necessary change is to add back the newline to the end of the file.

**Comment by ahmaxed** [comment]
Thank you for the review.

**Review by moT01 — APPROVED** [review]
LGTM 🎉

**Pull request merged**